src/SignInWithKlarna.php

```diff
@@ -24,7 +25,7 @@ class SignInWithKlarna {
 	 *
 	 * @var string
 	 */
-	public static $placement_hook = 'siwk_placement';
+	public static $placement_hook = 'output_button';
```

Comment:
This is used by a did_action on row 158. Is this a existing hook we want to use from WordPress or WooCommerce? And can we be sure its not used by other plugins if we limit it to only allow it to trigger once?